streamer/output_stream.py

```diff
@@ -60,16 +59,44 @@ def is_dash_only(self) -> bool:
     """Returns True if the output format is restricted to DASH protocol"""
     assert self.codec is not None
     return self.codec.get_output_format() is 'webm'
+  
+  def get_init_seg_file(self, **kwargs) -> Pipe:
+    INIT_SEGMENT = {
+      MediaType.AUDIO: '{dir}/audio_{language}_{channels}c_{bitrate}_{codec}_init.{format}',
+      MediaType.VIDEO: '{dir}/video_{resolution_name}_{bitrate}_{codec}_init.{format}',
+      MediaType.TEXT: '{dir}/text_{language}_init.{format}',
+    }
+    path_templ = INIT_SEGMENT[self.type].format(**self._features, **kwargs)
+    return Pipe.create_file_pipe(path_templ, mode='w')
+
+  def get_media_seg_file(self, **kwargs) -> Pipe:
+    MEDIA_SEGMENT = {
```

Comment:
I like that you decided to move these here.  I imagine this will make some concat tasks easier, since you can get the output path.